src/TConsole.h

```diff
@@ -325,6 +325,10 @@ public slots:
     // profiles - asynchronously - to load in an updated map
     void slot_reloadMap(QList<QString>);
 
+protected:
+    void dragEnterEvent(QDragEnterEvent *e);
```

Comment:
Minor code style detail - the argument name `e` is not needed here (and in the next one) and there shouldn't be a space between the type `QDragEnterEvent` and the pointer indicator `*`.

Reply:
I think it's good to have the argument anyhow, even if it's not needed - it helps the readability.